### PR TITLE
Add single Lesson Plan page - rebased

### PR DESCRIPTION
**Pull request by outdoor2kode**
Same as #2540, but rebase to the trunk.
New commits in this PR start from [Rename to sidebar-meta-list](https://github.com/WordPress/Learn/pull/2645/commits/b3f91e3cced61f8c972262b4da5017e9552cfbf1)

## Screenshot

image

## Testing

1. Check single lesson plan page functions
2. Check single tutorial page sidebar
3. Check single course page sidebar
4. Make sure everything works as expected

### CODE REVIEW

File: wp-content/themes/pub/wporg-learn-2024/patterns/sidebar-tutorial.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * Slug: wporg-learn-2024/sidebar-tutorial
+ * Inserter: no
+ *
+ * @package wporg-learn-2024
+ */
+
+use function WPOrg_Learn\Sensei\{get_my_courses_page_url};
+
+$completed_course = Sensei_Utils::user_completed_course( get_the_ID() );
+$current_post     = get_post();
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2024/patterns/sidebar-tutorial.php]: I think we can remove all of this
```suggestion
```

**Comment by adamwoodnz** [comment]
Looks like the print button should be primary (solid) unless there are slides

[Design](https://www.figma.com/design/ZKaf6u8mIHkAanluWafXAp/Learn?node-id=4319-12645&m=dev):
Screenshot 2024-07-09 at 9 49 51 PM

Sandbox:
Screenshot 2024-07-09 at 9 47 22 PM

**Comment by adamwoodnz** [comment]
The local nav is showing the first Lesson Plan title

Screenshot 2024-07-09 at 9 53 38 PM

**Review by adamwoodnz — COMMENTED** [review]
Nearly there! Couple of comments inline

File: wp-content/themes/pub/wporg-learn-2024/patterns/sidebar-tutorial.php (hunk 1 above)
Comment (outdoor2kode) [reply wp-content/themes/pub/wporg-learn-2024/patterns/sidebar-tutorial.php]: 7c7ef48

**Comment by outdoor2kode** [comment]
> Looks like the print button should be primary (solid) unless there are slides

missed that one! thanks. 134dd68

| | |
|-|-|
| **No slides** | image |
| **Viewable & Downloadable** | image |
| **Viewable only** | image |
| **Downloadable only** | image |

**Comment by outdoor2kode** [comment]
> The local nav is showing the first Lesson Plan title

👍👍 8caf02d

|| Default | Scroll |
|-|-|-|
| **L2B**  <br> Archive Lesson Plans | image | Same |
| **L3** <br> Single Lesson Plans | image | image |

**Review by adamwoodnz — APPROVED** [review]
LGTM! I'm going to merge and ship so I can continue to iterate on the sidebar.

**Pull request merged**